OpenRA.Game/CPos.cs

```diff
@@ -19,23 +19,22 @@ namespace OpenRA
 	public struct CPos : IScriptBindable, ILuaAdditionBinding, ILuaSubtractionBinding, ILuaEqualityBinding, ILuaTableBinding, IEquatable<CPos>
 	{
 		public readonly int X, Y;
+		public readonly int Layer;
```

Comment:
I assume that the runtime will be aligning `CPos` instances on 32 or 64 bit boundaries, so making `Layer` a byte by itself shouldn't change anything... right?  We'd need to make X and Y shorts to win anything, but then it doesn't really matter what we make Layer (a byte would probably make sense though).